dali/operators/random/noise/salt_and_pepper_noise.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/random/noise/salt_and_pepper_noise.h"
+#include "dali/operators/random/rng_base_cpu.h"
+#include "dali/pipeline/operator/arg_helper.h"
+
+namespace dali {
+
+DALI_SCHEMA(noise__SaltAndPepper)
```

Comment:
Since it doesn't support real "salt" (white) and "pepper" (black) noise, but operaters per-channel instead, perhaps it should be more accurately described as ImpulseNoise?

Reply:
it does support it now.